### DISCUSSION TITLE
第2节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#66] [笔记]
请在下方提交第 2 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by xiaomile** [comment]
[轻松玩转书生·浦语大模型趣味Demo](https://blog.csdn.net/weixin_42567071/article/details/135423120)

**Comment by liuxianyi** [comment]
[趣味Demo](https://liuxianyi.notion.site/Demo-34ea43b2fbf84250af5c9406b7eaacc9?pvs=4)

**Comment by li-chuanwei** [comment]
https://www.yuque.com/epiphany-erkzt/epcy7g/tenat8n3u4rd52wz?singleDoc# 《02 轻松玩转书生·浦语大模型趣味 Demo》
最近要考试了😭😭😭

**Comment by GingerNg** [comment]
https://note.youdao.com/s/9qXSHATQ

**Comment by cookieswolf** [comment]
[02 轻松玩转书生·浦语大模型趣味 Demo](https://www.mubu.com/doc/6yzyruDiUAO) 笔记作业都在里面

**Comment by ShelterWFF** [comment]
class2: http://t.csdnimg.cn/NKD1b

**Comment by blissSeven** [comment]
2 轻松玩转书生·浦语大模型趣味Demo
https://github.com/blissSeven/knowledge/blob/master/LLM/2%20%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%B6%A3%E5%91%B3Demo.md

**Comment by zqp1226358** [comment]
[Note](https://www.yuque.com/kongfuzijiadeqiu/azzns0/xy6ik1kfzequ7pyx/edit?toc_node_uuid=9cm-IbB3QSaZn4EC)

**Comment by KJ-Huang-sz** [comment]
第二次课笔记：https://www.bilibili.com/read/cv29126227/?jump_opus=1

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135439314

**Comment by i-am-so-lucky-today** [comment]
第二次课堂笔记：https://blog.csdn.net/weixin_49534890/article/details/135580251?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135580251%22%2C%22source%22%3A%22weixin_49534890%22%7D

**Comment by leonnapsun** [comment]
第二课笔记：
https://zhuanlan.zhihu.com/p/676511573

**Comment by nini0919** [comment]
This is my note for class2:
https://morning-gaura-eed.notion.site/02-37138b4c19e440d0acae33846c3f4345?pvs=4

**Comment by hahahamamama** [comment]
第二次笔记：https://github.com/hahahamamama/InternLM/edit/main/Day2%20Notes.md

**Comment by haizeigh** [comment]
第二次笔记 https://note.youdao.com/s/FornwD8O
玩转demo https://www.bilibili.com/video/BV1Ci4y1z72H/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/676633156

**Comment by xingyuan3** [comment]
class2：https://www.mubu.com/doc/1MU_2zYPPxd

**Comment by AIzealotwu** [comment]
第二节课笔记如下：
https://blog.csdn.net/m0_72806612/article/details/135458668?spm=1001.2014.3001.5502

**Comment by Hyt560** [comment]
第二节笔记：https://blog.csdn.net/Hyt_foxxx/article/details/135396145

**Comment by wangxuejiao** [comment]
第二节笔记：https://zhuanlan.zhihu.com/p/676586122

**Comment by dragonzzl** [comment]
第二节笔记：https://github.com/dragonzzl/works_internlm/blob/main/works/work02.md

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/2.%20internlm%20demo

**Comment by zhangzhidaSunny** [comment]
第二节笔记：https://blog.csdn.net/m0_65719612/article/details/135580446
